package.json

```diff
@@ -6,20 +6,9 @@
   },
   "description": "JavaScript ROOT",
   "homepage": "https://root.cern/js/",
-  "type": "module",
-  "main": "./build/jsroot.js",
   "module": "./modules/main.mjs",
-  "exports": {
-    ".": "./modules/main.mjs",
-    "./core": "./modules/core.mjs",
-    "./draw": "./modules/draw.mjs",
-    "./io": "./modules/io.mjs",
-    "./tree": "./modules/tree.mjs",
-    "./colors": "./modules/base/colors.mjs",
-    "./hierarchy": "./modules/gui/HierarchyPainter.mjs",
-    "./latex": "./modules/base/latex.mjs",
-    "./geom": "./modules/geom/TGeoPainter.mjs"
-  },
+  "main": "./modules/main.mjs",
+  "browser": "./build/jsroot.js",
```

Comment:
When `"browser"` is used?

Reply:
https://docs.npmjs.com/cli/v8/configuring-npm/package-json#browser